tests/test_integration_all_rust.py

```diff
@@ -300,19 +292,21 @@ def send_notification(self, channel=None, version=None, data=None,
 
         log.debug("%s body: %s", method, body)
         log.debug("  headers: %s", headers)
-        http.request(method, url.path.encode("utf-8"), body, headers)
-        resp = http.getresponse()
-        log.debug("%s Response (%s): %s", method, resp.status, resp.read())
-        http.close()
-        assert resp.status == status, \
-            "Expected %d, got %d" % (status, resp.status)
+        resp = requests.request(
```

Comment:
This pattern happens a lot. Switched to using `requests` instead of the older `http` library.